tests/test_hyperbolic_space.py

```diff
@@ -110,12 +105,11 @@ def test_log_and_exp_general_case(self):
         base_point = gs.array([4.0, 1., 3.0, math.sqrt(5.)])
         point = gs.array([2.0, 1.0, 1.0, 1.0])
```

Comment:
maybe also call this expected - in general names like point are pretty bad in terms of naming